code/modules/antagonists/heretic/knowledge/sacrifice_knowledge/sacrifice_buff.dm

```diff
@@ -109,9 +109,9 @@
 	bloodiest_wound.adjust_blood_flow(-0.5 * seconds_between_ticks)
 
 /// Torment the target with a frightening hand
-/proc/fire_curse_hand(mob/living/carbon/victim, turf/forced_turf)
+/proc/fire_curse_hand(mob/living/carbon/victim, turf/forced_turf, range = 8)
```

Comment:
`fire_curse_hand` spawns `/obj/projectile/curse_hand/hel`

The status effect spawns `/obj/projectile/curse_hand`

You will need to address that (they're not exactly identical)